### PR TITLE
Moved to typescript and updated react-scripts

### PR DESCRIPTION
**Pull request by viniciussalvati**
I still need to add lint config, but I'm working now, and with this, we can start working.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,9 +3,17 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "react": "^16.8.6",
-    "react-dom": "^16.8.6",
-    "react-scripts": "3.0.1"
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.5.0",
+    "@testing-library/user-event": "^7.2.1",
+    "@types/jest": "^24.9.1",
```
Comment (pedrofracassi) [inline package.json:9]: Shouldn't these be `devDependencies` instead?

**Review by pedrofracassi — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (viniciusmelquiades) [reply package.json:9]: This is how the `create-react-app` generated it, but yes, that makes sense.

Comment (viniciussalvati) [reply package.json:9]: It should, if it was a backend project, where we can restore only some dependencies. But for a `create-react-app`, it doesn't make any difference, as everything is a devDependency. In the end, webpack will compile only what's being referenced by our app.

Comment (pedrofracassi) [reply package.json:9]: Cool!

**Review by bellesamways — APPROVED** [review]

**Pull request merged**